CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+<!--
+(c) Copyright IBM Corp. 2017 All Rights Reserved
```

Comment:
Please use 2017, 2017. I think the copyright check needs this.

Reply:
There are no copyright checks on this repo as of yet, but I don't have an issue with that modification.  This was only left marked as a WIP because I thought there would be further suggestions made, but additions can always be made after an initial commit.

Reply:
There also needs to be a space after the comma.